modules/dcache/src/main/java/org/dcache/services/httpd/HttpdRequestLog.java

```diff
@@ -27,11 +28,23 @@
 class HttpdRequestLog extends AbstractLifeCycle
     implements RequestLog
 {
-    private static final Logger LOGGER
-            = LoggerFactory.getLogger(HttpdRequestLog.class);
+    //private static final Logger LOGGER = LoggerFactory.getLogger(HttpdRequestLog.class);
+
+
+    private static final Logger ACCESS_LOGGER = LoggerFactory.getLogger("org.dcache.services.httpd");
+
 
     public void log(Request request, Response response)
     {
-        LOGGER.trace("request: {}; response: {}", request, response);
+        if(ACCESS_LOGGER.isInfoEnabled()){
+            NetLoggerBuilder log = new NetLoggerBuilder(NetLoggerBuilder.Level.TRACE, "org.dcache.services.httpRequestLog").omitNullValues();
```

Comment:
use static import of NetLoggerBuilder.Level.TRACE; reduces the verbosity of the code.

Change the NetLoggerBuilderEvent to org.dcache.httpd.request